packages/components/src/components/index.tsx

```diff
@@ -85,20 +86,7 @@ export const CourierComponents: React.FunctionComponent = () => {
         toast: Boolean(toastElement),
       }}
     >
-      {inboxElement &&
-        createPortal(
```

Comment:
do we want to get rid of this?

Reply:
Yeah. It's fine. That repo is more of a playground than anything.